### PR TITLE
Add new Port: NetSurf Web Browser

### PR DESCRIPTION
**Pull request by Cebion**
NetSurf

Game Information

Title: NetSurf
URL: [https://www.asahi-net.or.jp/~cs8k-cyu/windows/a7xpg_e.html](https://github.com/tailtwo/netsurf)

Onscreen keyboard can be accessed on the bottom right corner (A) symbol.

CFW Tests
[X] AmberELEC
[X] ArkOS
[X] ROCKNIX
[X] muOS

[X] TSP Crossmix (Optional)
[]  Knulli (Optional) 

Resolution Tests
Test all major resolutions:
[] 480x320 (Optional)
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/netsurf/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Notes
+Thanks to the [Netsurf Team] (https://www.netsurf-browser.org/) for creating this browser.
```
Comment (kloptops) [inline ports/netsurf/README.md:2]: Syntax error in markdown:
```suggestion
Thanks to the [Netsurf Team](https://www.netsurf-browser.org/) for creating this browser.
```

**Review by kloptops — CHANGES_REQUESTED** [review]

File: ports/netsurf/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## Controls
+
+The following instructions are for a right-facing character. 
+
+| Button | Action |
+|--|--| 
+|Left Analogue Stick | Move Mouse|
+|A| Left Mouse |
+|B| Slow Mouse |
+| HK + X| Refresh (F5)
```
Comment (cdeletre) [inline ports/netsurf/README.md:16]: markdown syntax error, missing a `|` a the end

**Review by cdeletre — CHANGES_REQUESTED** [review]
Just one markdown syntax error in README.md :)

**Comment by helio-marcos** [comment]
hello friend! Thank you for your contributions to PortMaster!
I added an option to Netsurf.sh to work on Arkos-Aeolux, the CFW_NAME" = " AeUX"
netsurf

**Pull request merged**